integration_test/workflow_tests/README.md

```diff
@@ -3,6 +3,13 @@
 Visit the [Developer Guide](https://nasa.github.io/cumulus-orca/docs/developer/development-guide/code/contrib-code-intro)
 for additional information on environment setup and [running/creating tests](https://nasa.github.io/cumulus-orca/docs/developer/development-guide/code/integration-tests).
 
+## Ingest Integration Test in Bamboo
+The `ORCA Deploy Plan` under ORCA project in Bamboo includes the ingest integration test which will run automatically once ORCA deployment is successful. Make sure to add the following three variables as well while running the deploy plan.
+
+- ORCA_RECOVERY_BUCKET_NAME
+- ORCA_COPY_TO_ARCHIVE_STEP_FUNCTION_ARN
+- ORCA_API_DEPLOYMENT_INVOKE_URL
```

Comment:
Recommend you just link to the appropriate section in the versioning-releases webpage to avoid redundant docs.

Reply:
Added

Reply:
This works. You can remove the variable stuff if you want. Removes the need to come back here once we automate the Bamboo variable handling.

Reply:
Removed from here.